modules/programs/chromium.nix

```diff
@@ -220,15 +220,22 @@ let
       };
 
     in
+
     lib.mkIf cfg.enable {
-      programs.${browser}.finalPackage = lib.mkIf (cfg.package != null) (
+      assertions = [
+        {
+          assertion = !(cfg.package == null && cfg.commandLineArgs != [ ]);
+          message = "Cannot set `commandLineArgs` when `package` is null for ${browser}.";
+        }
+      ];
```

Comment:
💭  Previously setting both was allowed, but was silently ignored. Happy to switch back if others desire.